.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -0,0 +1,139 @@
+        2. Connect to a remote server via SSH.
+        3. Try to execute `clear`.
+        4. Observe `xterm-ghostty` error message above.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Ghostty Version
+      description: Paste the output of `ghostty +version` here.
+      placeholder: |
+        Ghostty 1.1.3
+
+        Version
+          - version: 1.1.3
+          - channel: stable
+        Build Config
+          - Zig version: 0.13.0
+          - build mode : builtin.OptimizeMode.ReleaseFast
+          - app runtime: apprt.Runtime.none
+          - font engine: font.main.Backend.coretext
+          - renderer   : renderer.Metal
+          - libxev     : main.Backend.kqueue
+      render: text
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: OS Version Information
+      description: |
+        Please tell us what operating system (name and version) you are using.
+      placeholder: Ubuntu 24.04.1 (Noble Numbat)
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: (Linux only) Display Server
+      description: |
+        If you run Linux, please tell us if you use X11 or Wayland. If you aren't sure, you can determine this by running `[ -z "$WAYLAND_DISPLAY" ] && echo X11 || echo Wayland`.
+      options:
+        - X11
+        - Wayland
+        - Other
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: (Linux only) Desktop Environment/Window Manager
+      description: |
+        If you run Linux, please tell us what Desktop Environment/Window Manager you are using (include the name and version).
+      placeholder: GNOME 47.4
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Ghostty Configuration
+      description: |
+        Please paste the output of `ghostty +show-config` here.
+      placeholder: |
+        font-family = CommitMono Nerd Font
+        font-family-bold = CommitMono Nerd Font
+        font-family-italic = CommitMono Nerd Font
+        font-family-bold-italic = CommitMono Nerd Font
+        font-feature = +cv07
+        font-size = 16
+        font-thicken = true
+        theme = catppuccin-mocha
+      render: ini
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional Relevant Configuration
+      description: |
+        If your issue involves other programs, tools, or applications in addition to Ghostty (e.g. Neovim, tmux, Zellij, etc.), please provide any relevant configuration information here.  If you use custom CSS or shaders for Ghostty, please also include them here, if applicable to your issue.
+      placeholder: |
+        `tmux.conf`
+        ---
+        set  -g default-terminal "tmux-256color"
+        set-option -sa terminal-overrides ",xterm*:Tc"
+        set  -g base-index      1
+        setw -g pane-base-index 1
+      render: text
```

Should this be in a code block, since `render` is `text`?  Or does `text` mean monospace?
```suggestion
      placeholder: |
        `tmux.conf`
        ---
        ```tmux
        set  -g default-terminal "tmux-256color"
        set-option -sa terminal-overrides ",xterm*:Tc"
        set  -g base-index      1
        setw -g pane-base-index 1
        ```
      render: text
```

---

> `render` (String): If a value is provided, user-submitted text will be formatted into a codeblock automatically.

The short answer is GitHub does some magic in the backend to convert it into a code block, if you add the backticks it displays them in the rendered field.